config/initializers/money.rb

```diff
@@ -0,0 +1,115 @@
+# encoding : utf-8
+
+MoneyRails.configure do |config|
+
+  # To set the default currency
+  #
+  # config.default_currency = :usd
+
+  # Set default bank object
+  #
+  # Example:
+  # config.default_bank = EuCentralBank.new
```

Comment:
This will be updated when using `oxr` or similar gem. Use the gem that I have recommended and manybe have a look at the video that I shared.

Reply:
Made correct changes in initialiser file and using money exchange rates gem